ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -2674,6 +2674,7 @@ periodics:
                 HYPERSHIFT_NODE_COUNT: 2
                 HYPERSHIFT_ZONES: "us-east-1a"
                 HYPERSHIFT_NAMESPACE: "cluster-bot"
+                HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT: "true"
```

Comment:
Shouldn't we also set this var for our other jobs like the presubmits?

Reply:
sgtm, I was wanting to merge this one first, test it, then extend it

Reply:
Actually I'm thinking we can change it here for some or the others and the reharse would tell us